test/unittest/statistics/dds/StatisticsQosTests.cpp

```diff
@@ -35,17 +38,15 @@ namespace dds {
  */
 TEST(StatisticsQosTests, StatisticsDataWriterQosTest)
 {
-    // TODO(jlbueno) Remove this guards after implementation. Here to prevent failures in current CI.
-#ifdef FASTDDS_STATISTICS
-    logError(STATISTICS_QOS_TEST, "This test is going to fail because API is not yet implemented.")
-
     EXPECT_TRUE(STATISTICS_DATAWRITER_QOS.reliability().kind == eprosima::fastdds::dds::RELIABLE_RELIABILITY_QOS);
     EXPECT_TRUE(STATISTICS_DATAWRITER_QOS.durability().kind == eprosima::fastdds::dds::TRANSIENT_LOCAL_DURABILITY_QOS);
-    // TODO(jlbueno) Pull mode is not yet exposed in DDS API
+    const std::string* pushMode_property = eprosima::fastrtps::rtps::PropertyPolicyHelper::find_property(
```

Comment:
Do we have an alias for `PropertyPolicyHelper` on namespace `eprosima::fastdds::rtps` ?

Reply:
It does not seem to be defined in the project. All the files that are using this class have `using eprosima::fastrtps::rtps` directive. The header file where this class is defined is `fastdds/rtps/attributes/PropertyPolicy.h`. Even though this file is within the `fastdds` include folder, the namespace defined is `eprosima::fastrtps::rtps`. Maybe this was done to keep consistency with the `rtps` layer which still keeps using the `eprosima::fastrtps::rtps` namespace.